trt13/parser_test.go

```diff
@@ -0,0 +1,276 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+
+	storage "github.com/dadosjusbr/storage"
```

Comment:
Não precisa de indenticador explítico para esse pacote. Ele já se chama storage (https://medium.com/golangspec/import-declarations-in-go-8de0fd3ae8ff)